i18n/en.json

```diff
@@ -3892,7 +3892,7 @@
   },
   {
     "id": "ent.compliance.csv.metadata.json.zipfile.appError",
-    "translation": "Unable to create the zip file"
+    "translation": ""
```

Comment:
Not sure where this translation string went

Reply:
This is because this PR uses an outdated enterprise codebase. I added it in b9a62905476fbbb4c9753bfb250e4a974023dbdd. Let's bring it back.

Reply:
@agnivade - this is a PR to a feature branch. To solve this problem, branch needs to be synced with master. I'm not fixing this, will be fixed before merging the feature branch